Client/src/main/java/de/baleipzig/pdfextraction/client/utils/PDFPreview.java

```diff
@@ -9,12 +9,14 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Optional;
 
 
 public class PDFPreview {
 
     private static final PDFPreview instance = new PDFPreview();
-    private Path pdfPath;
+
+    private RendererWrapper wrapper;
     private int numberOfPages = 0;
```

Comment:
Wozu brauchen wir den Wrapper ?

Reply:
also du hast ja in der klasse ziemlich viel auf den Wrapper angepasst, aber welchen sinn hat der ?